scorewidget/build.gradle.kts

```diff
@@ -34,19 +34,30 @@ android {
     kotlinOptions {
         jvmTarget = "17"
     }
-
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = libs.versions.composeCompiler.get()
+    }
     buildFeatures {
         viewBinding = true
     }
 }
 
 dependencies {
-
-    implementation(libraries.androidx.core.ktx)
-    implementation(libraries.androidx.appcompat)
-    implementation(libraries.android.material)
+    implementation(libraries.androidx.compose.activity)
+    implementation(platform(libs.androidx.compose.bom))
+    implementation(libs.androidx.compose.ui)
+    implementation(libs.androidx.compose.ui.graphics)
+    implementation(libs.androidx.compose.ui.tooling)
+    implementation(libs.androidx.compose.ui.tooling.preview)
+    implementation(libs.androidx.compose.material3)
+    implementation(libs.androidx.compose.icons.extended)
     implementation(libraries.arklib)
     implementation(libraries.orbit.mvi.viewmodel)
+    implementation(libs.orbit.mvi.compose)
+    implementation("com.github.mdrlzy:ComposeCounterSlider:0.1.4")
```

Comment:
Can this dependency be added in predefined version management scheme that we already have?